.delete_game.php

```diff
@@ -3,20 +3,17 @@
   session_id($_POST['session_id']);
   session_start();
 
-  require_once ".db_config.php";
+  require_once ".functions.php";
+  require_once ".gcal.php";
 
-  $connection = new PDO("mysql:host=$dhost;dbname=$dname", $duser, $dpassword);
-  $connection->setAttribute(PDO::ATTR_ERRMODE, PDO::ERRMODE_EXCEPTION);
-
-  if(!isset($dname)) {
-    $dname = 'ambulanc_web';
-  }
+  $calendar = new GoogleCalendar();
```

Comment:
Can we rename this variable to make it clear that this is a Google calendar object? It works fine, but, for instance, [line 28](https://github.com/rpiambulance/website/blob/189ca66e628103e9c1bd127c6d560bd7388deb6d/.delete_game.php#L28) isn't clear that we're deleting the event from Google.